roombasim/graphics/display.py

```diff
@@ -57,16 +66,46 @@ def on_draw(self):
 
         Display._draw_gridlines()
 
+        self._draw_header(self.environment.good_exits,
+                          self.environment.bad_exits,
+                          self.environment.score,
+                          int(self._elapsed / 1000))
+
         for r in self.environment.roombas:
             if isinstance(r, roomba.TargetRoomba):
-                Display._draw_target_roomba(r)
+                if self.environment.target_roomba == r.tag:
+                    Display._draw_target_roomba(r,
+                                                self.environment.target_type)
+                else:
+                    Display._draw_target_roomba(r)
             else:
                 Display._draw_obstacle_roomba(r)
 
         Display._draw_drone(self.environment.agent)
 
+    def on_mouse_release(self, x, y, button, modifiers):
```

Comment:
Andrew said that roombas were hard to click on, maybe increase the leeway by adding a small constant to the roomba_radius.

Reply:
Fixed